plugins/google-cloud/tests/storage.spec.ts

```diff
@@ -73,10 +73,10 @@ describe('Google Cloud Storage', () => {
   // storage test
 
   describe('Google Cloud Storage', () => {
-    const createPackage = (cloudDatabase: ITestLocalData, name: string, done: jest.DoneCallback) => {
+    const createPackage = (_1: ITestLocalData, _2: string, done: jest.DoneCallback): void => {
```

Comment:
here `_cloudDatabase` also works.  Why numbers?

Reply:
Typescript accepts `_XXX` variable name when is not used.

Reply:
But ESLint will not say that the variables are unused?

Reply:
Well, ESLint don't say anything about because they are not the latest parameter of the function. I'll revert every match